05/Menu.js

```diff
@@ -0,0 +1,15 @@
+import React, { useLayoutEffect } from 'react';
+import MenuItem from './MenuItem';
+// const arr = [<li>item1</li>, <li>item2</li>];
+// const jsx = <ul>{arr}</ul>;
+const Menu = props => {
+	// zbieram elementy MenuItem z wlasciwoscami przechwyconymi z argumentu(props)(obiekt items z komponentu Nav) pozniej tworze tablice na podstawie tych wlascwiosci, ale wlasciwosci wrzucam do "argumentu?" MenuItem i przechowuje cala tablice gotwoych MenuItemó'w z odpowiednimi wlasciwoscami podanymi w Nav pod zmienna jsx, ktora pozniej umieszczam w ul liscie,
```

Comment:
Jeśli przekazujemy dane do komponentu to raczej mówimy o `props` - ale jak się za niedługo okaże to każdy komponent  może być funkcję. Co powoduje, że props to faktycznie argument/parametr  o tej właśnie nazwie :)